fetcher/src/server.ts

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import "express-async-errors";
+require("dotenv").config();
+
+import reportsHandler from "handlers/reports-handler";
+
+const app = express();
+const port = parseInt(process.env.PORT);
+
+app.get("/reports", reportsHandler);
```

Comment:
We need a grpahql integration so we could query the service for paramerters: quarters, years, fund bodies etc.etc.

Reply:
Moved the entire thing to Graphql